nuitka/distutils/DistutilCommands.py

```diff
@@ -231,6 +232,18 @@ def _build(self, build_lib):
                     if fullpath.lower().endswith((".py", ".pyw", ".pyc", ".pyo")):
                         os.unlink(fullpath)
 
+            # If the Python module has more than one parent package (e.g.
+            # 'a.b.mod'), the compiled module will be in 'a.b/mod.so'. Move it
+            # to 'a/b/mod.so', to make imports work.
+            if package and "." in package:
+                compiled_package_path = os.path.join(build_lib, package)
+                assert os.path.isdir(compiled_package_path), compiled_package_path
+
+                parts = package.split(".")
+                fixed_package_path = os.path.join(build_lib, *parts)
+                dir_util.copy_tree(compiled_package_path, fixed_package_path)
+                dir_util.remove_tree(compiled_package_path)
```

Comment:
Is this not really a move then?

Reply:
Yes, it is a move. This was just the most succinct way to write this. `shutil.move` doesn't cut it, since the details are a bit unusual:

- I want to move all contents of `compiled_package_path` into `fixed_package_path`
- I want to create `fixed_package_path`, if it doesn't already exist

Is there something in `nuitka.utils.FileOperation` that does this?

Reply:
I moved to `FileOperation` instead of `dir_util`, but there isn't a function there to do a "move all the contents of this directory into this directory" either, so it's still a copy + remove.